definitions/scenarios/foreman_upgrade.rb

```diff
@@ -0,0 +1,145 @@
+module Scenarios::ForemanUpgrade
+  class Abstract < ForemanMaintain::Scenario
+    def self.upgrade_metadata(&block)
+      metadata do
+        tags :upgrade_scenario
+        confine do
+          feature(:foreman_install)
```

Comment:
you dropped the `|| ForemanMaintain.upgrade_in_progress == 'nightly'` is that because the "foreman_install" feature will always be there?

Reply:
Yea